packages/platform-android/src/commands/runAndroid/runOnEmulator.ts

```diff
@@ -109,4 +140,4 @@ function createInstallError(error: Error & {stderr: string}) {
   return new CLIError(`Failed to install the app. ${message}.`, error);
 }
 
-export default runOnAllDevices;
+export default runOnEmulator;
```

Comment:
I think we still `runOnAllDevices`, not just on a single "emulator" - all in all you have `forEach` inside on line 110.